lib/galaxy/dependencies/__init__.py

```diff
@@ -273,6 +286,9 @@ def check_custos_sdk(self):
     def check_hvac(self):
         return "hashicorp" == self.vault_type
 
+    def check_pkce(self):
+        return True
```

Comment:
Should this be checking `oidc_backends_config` prior to returning True, or is this just a placeholder for now?

Reply:
Good catch; yeah, that was a placeholder while I worked out the parsing above.  It should just use `self.pkce_support`